slapp/transforms/transform_pipeline.py

```diff
@@ -170,25 +171,38 @@ def run(self, db_conn: query_utils.DbConnection):
 
         playback_fps = self.args['output_fps'] * self.args['playback_factor']
 
+        # experiment-level artifact
+        full_video_path = output_dir / "full_video.webm"
+        transform_to_webm(
+                downsampled_video, str(full_video_path),
+                playback_fps, self.args['webm_bitrate'])
+
         # create the per-ROI artifacts
         insert_statements = []
         for roi in rois:
             # mask and outline from ROI class
             mask_path = output_dir / f"mask_{roi.roi_id}.png"
             outline_path = output_dir / f"outline_{roi.roi_id}.png"
+            full_outline_path = output_dir / f"full_outline_{roi.roi_id}.png"
             sub_video_path = output_dir / f"video_{roi.roi_id}.webm"
             max_proj_path = output_dir / f"max_{roi.roi_id}.png"
             avg_proj_path = output_dir / f"avg_{roi.roi_id}.png"
             trace_path = output_dir / f"trace_{roi.roi_id}.json"
 
             mask = roi.generate_ROI_mask(
                     shape=self.args['cropped_shape'])
+            mask = np.uint8(mask * 255 / mask.max())
```

Comment:
I think we should normalize against the global roi_mask max or video max, so that from one roi to another there is a difference in intensity in the mask. ie: An roi with a max probability of 0.95 and one with a max probability of 0.7 will both end up having those pixels at 255, which I think is less informative to the viewer. I think we lose some knowledge in that transform.

Reply:
These are not probabilities, they are weights. I made this choice intentionally to make sure that the masks were visible on the scale. These users have not usually seen weighted masks, so I don't expect them to be able to parse the weight value and have that contribute to their judgment on the ROI. The mask weights in the image are valuable if they see that the low-weighted (darker areas) correspond with some pixels that don't appear to be part of the cell.

Reply:
I'd also mention that the mask was already being normalized before this change by `imageio.imsave()` but it was throwing an annoying warning that `you can get rid of this warning by doing the uint8 conversion yourself`, so I just added this to suppress that output.